docs/docs/reference/release-notes/image-migration-guide.md

```diff
@@ -0,0 +1,234 @@
+
+Note that you can use both packages at the same time and may need to in situations where you're using a gatsby-image compatible CMS as a data source and that CMS plugin has not yet updated to a version compatible with the new API.
+
+If you're looking for other documentation on the new plugin, see:
+
+- How to guide: //todo
+- Reference guide: //todo
+
+## What changed?
+
+### New Syntax
+
+The new plugin requires significant syntax changes. We've provided a [codemod](#codemod-instructions) to help you migrate, but this section will explain the changes.
+
+#### Import change
+
+Previously, `GatsbyImage` was the default export from `gatsby-image`. This is no longer the case.
+
+```javascript
+// import Img from "gatsby-image"
+import { GatsbyImage } from "gatsby-plugin-image"
+```
+
+#### GraphQL changes
+
+This is one of the bigger changes. There are no more fragments to use, instead things like layout and format are passed as arguments to the resolver.
+
+This is the previous syntax.
+
+```javascript
+import { graphql } from "gatsby"
+
+export const query = graphql`
+  {
+    file(relativePath: { eq: "images/example.jpg" }) {
+      childImageSharp {
+        fixed {
+          ...GatsbyImageSharpFixed
+        }
+      }
+    }
+  }
+`
+```
+
+The new syntax looks like this. The fragment is removed in favor of `gatsbyImageData` which is used for all images. Previous differences, such as fixed vs. fluid, webP generation and placeholder type are passed as arguments to the resolver.
+
+```javascript
+import { graphql } from "gatsby"
+
+export const query = graphql`
+  {
+    file(relativePath: { eq: "images/example.jpg" }) {
+      childImageSharp {
+        gatsbyImageData(layout: FIXED)
+      }
+    }
+  }
+`
+```
+
+There are other changes to the available argument structure in the section on [API changes](#api-changes).
+
+#### Component changes
+
+The last change is to the JSX component. The import name is potentially different and the query result is also different.
+
+```javascript
+// import Img from "gatsby-image"
+import { GatsbyImage } from "gatsby-plugin-image"
+
+const HomePage = ({ data }) => {
+  return (
+    // <Img fixed={data.file.childImageSharp.fixed} />
+    <GatsbyImage image={data.file.childImageSharp.gatsbyImageData} />
+  )
+}
+```
+
+### API Changes
+
+In addition to the syntax changes for using gatsby-plugin-image, there are also changes to the API that affect the resolver arguments (and StaticImage).
+
+#### fluid
+
+The `fluid` image type has been deprecated in favor of two alternatives.
+
+The first is an image type called `fullWidth`. This image is designed to be used for things like hero images and generates image sizes accordingly. Instead of passing something like `maxWidth`, it takes an array called `breakpoints` that will generate images designed for those screen sizes. Note that `fullWidth` images will expand to fit the width of their container, even if that width is larger than the source image.
```

Comment:
```suggestion
The first is an image type called `fullWidth`. This image is designed to be used for things like hero images and generates image sizes accordingly. Instead of passing something like `maxWidth`, it takes an array called `breakpoints` that will generate images designed for those screen sizes. In most cases will not need to provide these, as there are default values that work for most sites. Note that `fullWidth` images will expand to fit the width of their container, even if that width is larger than the source image.
```